mobile/src/libs/components/switch/switch.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { RNSwitch, View } from '#libs/components/components';
```

Comment:
We can import RNSwitch directly from react-native for this file, no need to re-export it. Because that component will be used only in this file